package.json

```diff
@@ -32,11 +32,11 @@
     "start": "mocha --watch --recursive --colors"
   },
   "peerDependencies": {
-    "eslint": ">= 4.1.0",
+    "eslint": ">= 6.7.0",
```

Comment:
I suppose this should elicit a major version bump of this package. I wasn't sure if I should include the change in this file in this pull request or if that should just be a separate commit.